app/src/main/java/com/vultisig/wallet/presenter/keygen/KeygenFlowViewModel.kt

```diff
@@ -85,22 +85,9 @@ internal class KeygenFlowViewModel @Inject constructor(
 
     suspend fun setData(vaultId: String, context: Context) {
         // start mediator server
-        val allVaults = vaultRepository.getAll()
 
-        val vault = if (vaultId == Screen.KeygenFlow.DEFAULT_NEW_VAULT) {
-            var newVaultName = ""
-            var idx = 1
-            while (true) {
-                newVaultName = "New vault ${allVaults.size + idx}"
-                if (allVaults.find { it.name == newVaultName } == null) {
```

Comment:
Can you please keep these logic there , so the vault name can be optional for user?

Reply:
Done